api/serializers.py

```diff
@@ -169,6 +171,18 @@ def get_rating(self, event):
         rate = Rate.objects.filter(event=event).aggregate(Avg('rate'))
         return rate['rate__avg']
 
+    def get_current_user_rating(self, event):
+        request = self.context.get("request")
+        if request and not request.user.is_anonymous:
```

Comment:
shouldn't the context always have a request?

Reply:
AttributeError: 'NoneType' object has no attribute 'user'
- apparently not